### DISCUSSION TITLE
Why is this repository divided into two extensions, client and server?

### DISCUSSION THREAD

**Discussion by yuhaibohotmail** [quarkiverse/quarkus-openapi-generator#904] [Ideas]
As far as I understand, the main difference is that quarkus-openapi-generator is based on OpenAPI Generator, and quarkus-openapi-generator-server is based on Apicurio Codegen tool. They both have the ability to generate client code and server code. 

The code generated by quarkus-openapi-generator implements the server code as long as the generated API interface is implemented. 
The code generated by quarkus-openapi-generator-server only lacks annotations such as @RegisterRestClient.

But there are some differences, for example, quarkus-openapi-generator supports openapi 3.1, but quarkus-openapi-generator-server does not support it yet.

Therefore, I suggest expanding both two extensions to support client code and server code, and renaming two extensions.

**Comment by ricardozanini** [comment]
@yuhaibohotmail It's intended to have everything under Apicurio in the future. Since we are not there yet, we have these two options.
